{"version":"3.7.1","messages":["\nAn error occurred while loading ./spec/pry_debugging_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0m\u001b[32mdef\u001b[0m \u001b[1;34mplus_two\u001b[0m(num)\u001b[0m\n\u001b[31m  \t\u001b[0mnum + \u001b[1;34m2\u001b[0m\u001b[0m\n\u001b[31m  \t\u001b[0mnum\u001b[0m\n\u001b[31m  \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mTypeError:\u001b[0m\n\u001b[31m  no implicit conversion of nil into String\u001b[0m\n\u001b[36m# ./lib/pry_debugging.rb:1:in `<top (required)>'\u001b[0m\n\u001b[36m# ./spec/pry_debugging_spec.rb:2:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000429789,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":1},"summary_line":"0 examples, 0 failures, 1 error occurred outside of examples"}